h2o-docs/src/product/data-science/upliftdrf.rst

```diff
@@ -35,6 +36,40 @@ In a tree node the result value for a split is sum :math:`metric(P, Q) + metric(
 
 You can read more information about ``uplift_metric`` on parameter specification page: `uplift_metric <algo-params/uplift_metric.html>`__.
 
+Uplift tree and prediction
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Tree-based algorithm means in every tree, it takes information about treatment/control group assignment and information about response directly into a decision about splitting a node. So there is only one tree for every class and both groups - no separate trees for treatment groups data and control groups data. 
+
+The uplift score is the criterion to make a decision similar to the Gini coefficient in the standard decision tree. Every leaf in a tree holds two predictions, which are calculated based on a distribution of response between treatment and control group observations:
+
+- :math:`TP_cl = (TY_cl + 1) / (T_cl + 2)`
+- :math:`CP_cl = (CY_cl + 1) / (C_cl + 2)`
+
+where:
+
+- :math:`cl` leaf `l` of a tree for class `c`
+- :math:`T_cl` how many observations in a leaf are from the treatment group (how many data rows in a leaf have ``treatment_column`` label == 1) 
+- :math:`C_cl` how many observations in a leaf are from the control group (how many data rows in the leaf have ``treatment_column`` label == 0)
+- :math:`TY1_cl` how many observations in a leaf are from the treatment group and respond to the offer (how many data rows in the leaf have ``treatment_column`` label == 1 and ``response_column`` label == 1)
+- :math:`CY1_cl` how many observations in a leaf are from the control group and respond to the offer (how many data rows in the leaf have ``treatment_column`` label == 0 and ``response_column`` label == 1)
+- :math:`TP_cl` treatment prediction of a leaf
+- :math:`CP_cl` control prediction of a leaf
+
+The uplift score for the leaf is calculated as difference between treatment prediction and control prediction:
+
+:math:`uplift_score_cl = TP_cl - CP_cl`
+
+Higher uplift score means more observations from treatment group respond to the offer than from control group. Which means offered treatment has positive effect. The uplift score can be negative, if more observations from control group respond to the offer without treatment.
+
+Final prediction is calculated in the same way as in DRF algorithm. Predictions for each observation is collect from all trees from an ensemble and mean prediction is returned. 
+
+When the ``predict`` method is called on test data the result frame has these columns:
+
+- ``uplift_predict``: result uplift prediction score, which is calculated as :math:`p_y1_ct1 - p_y1_ct0`
```

Comment:
When I run uplift, I do not see this output:

- ``uplift_predict``: result uplift prediction score, which is calculated as :math:`p_y1_ct1 - p_y1_ct0`
- ``p_y1_ct1``: probability the response is 1 if the row is from the treatment group
- ``p_y1_ct0``: probability the response is 1 if the row is from the control group
- 
I see this:
<img width="294" alt="Screen Shot 2022-04-05 at 6 08 23 AM" src="https://user-images.githubusercontent.com/20548431/161760830-693b45cf-87b5-4629-b210-14f4616d87ac.png">

Reply:
Thanks, @meganjkurka, for reporting this issue; the bug is fixed here: https://github.com/h2oai/h2o-3/pull/6138. So then, this documentation is correct.